README.md

```diff
@@ -1,89 +1,59 @@
 # electronjs.org
 
-This repository contains the code for the [Electron](https://www.electronjs.org/) website. It is built using
-[Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+[![Push and publish main](https://github.com/electron/website/actions/workflows/push-main.yml/badge.svg)](https://github.com/electron/website/actions/workflows/push-main.yml)
+[![Update i18n deploy](https://github.com/electron/website/actions/workflows/update-i18n-deploy.yml/badge.svg)](https://github.com/electron/website/actions/workflows/update-i18n-deploy.yml)
+[![Crowdin](https://badges.crowdin.net/electron/localized.svg)](https://crowdin.com/project/electron)
+
+This repository contains the code for the [Electron](https://www.electronjs.org/) website.
+It is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
 
 ## Installation
 
+To get started with a local version of the website, install dependencies with `yarn`. To get the
+website running in dev mode, run `yarn start`. To get a production build of the website, run
+`yarn build`.
+
 ```console
-yarn install
+yarn
+yarn start
 ```
 
 ## How to modify the documentation
 
-For English documentation the changes need to happen upstream in [`electron/electron`][]. These
-changes are picked automatically as soon as they are done in the stable branch. The exception
-is when there is a new major release.
-
-The translation work happens in [Crowdin] and the changes are picked up periodically every
-15 minutes.
+### English docs
 
-Any documentation changes done manually in this repo will be eventually override by any of these
-updates.
+For English documentation, the documentation is mirrored from the [electron/electron][] repo.
+Changes are picked automatically by the [electron-website-updater][] webhook as soon as they
+are made the latest stable release branch.
```

Comment:
nit: I think we should use backticks around repo names, and be consistent with naming - if we refer to one as `electron/electron`, then it should also be `electron/electron-website-updater` for the other, or vice versa (with a bare `electron`).